<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="ten">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>91</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ten_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ten_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ten_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>40</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ten_6">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>290</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>60~</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ten_5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>290</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>631</width>
     <height>51</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;나이를 선택해주세요.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
